test/lib/server/services/qualityControlFlag/QcFlagTypeService.test.js

```diff
@@ -275,4 +278,84 @@ module.exports = () => {
             expect(flagTypes.map(({ id }) => id)).to.have.all.ordered.members([11, 12, 13]);
         });
     });
+
+    describe('Creating QC Flag Type', () => {
+        it('should successfuly create QC Flag Type', async () => {
+            const parameters = {
+                name: 'A',
+                method: 'AA+',
+                bad: false,
+                color: '#FFAA00',
+                externalUserId: 1,
+            };
+            const newQCFlag = await qcFlagTypeService.create(parameters);
+            delete parameters.externalUserId;
+            expectObjectToBeSuperset(newQCFlag, parameters);
```

Comment:
I think it would be more clear to explicitely extract and check properties, maybe using object destructuring

Reply:
ok, applied